cirrus/server/glean/metrics.yaml

```diff
@@ -0,0 +1,44 @@
+---
+$schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
+
+cirrus_events:
+  enrollment:
+    type: event
+    description: >
+      Recorded when a user has met the conditions and is first bucketed into an
+      experiment (i.e. targeting matched and they were randomized into a bucket
+      and branch of the experiment). Expected a maximum of once per experiment
+      per user.
+    extra_keys:
+      experiment:
+        type: string
+        description: The slug/unique identifier of the experiment
```

Comment:
Could we call this `experiment_slug` or something so that it is clear that this the identifier?

Reply:
@eliserichards We want to make sure the shape of this ping exactly matches the existing structure of the 'enrollment' ping sent by other Nimbus clients, so let's make sure that's the case.  @yashikakhurana can you link to the definition of the existing enrollment event sent by the SDK in Fenix/iOS?

Reply:
oh I was referring from here https://github.com/mozilla/application-services/blob/main/components/nimbus/metrics.yaml

Reply:
Awesome, that totally makes sense 👍